### ISSUE TITLE
[BUG REPORT] - Wifi dissapear on sleep

### ISSUE THREAD

**Issue by coffandro** [Atlas-OS/Atlas#945]
### Description

Whenever I let my laptop falls asleep wifi dissapears from the quick menu

### Steps to reproduce (add screenshots if applicable)

1. Let laptop fall asleep
2. Wake up laptop

### Expected behavior

Nothing happens and my menu keeps being there

### Actual behavior (add screenshots if applicable)

the wifi menu dissapears

### Atlas Version

Atlas 10 22H2

### Desktop information

- Atlas 11 23H2
- Asus TUF gaming f15 FX506HCB
- Core i5 11400H 11th gen 2.70GHz
- 24 gb of ram
- Intel Dual Band Wireless-AC 3165
- Intel(R) UHD Graphics
- NVIDIA GeForce RTX 3050

### Requisites

- [X] This is not a support issue or a question. For any support, questions or help, join our [Discord server](https://discord.atlasos.net).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I checked the [documentation](https://docs.atlasos.net) to understand that the issue I am reporting is not normal behavior.
- [X] I understand that not filling out this template correctly will lead to the issue being closed.

**Comment by Totallynotmwa** [comment]
1. Right click "Your internet icon".
2. Tab "Open Network & Internet Settings".
3. On left subject Settings, tab "Your internet".
4. On right subject related settigs, Tab "Change Adapter Options".
5. Right click your interfnet, then Tab Properties.
6. Tab "Configure", then Tab on Power Management tab.
7. Remove all checkmarks, this will ensure not to shutdown WiFi adapter to save power.

**Comment by Totallynotmwa** [comment]
This isnt a atlas related issue i think @coffandro

**Comment by coffandro** [comment]
That dosen't appear, I'm on windows 11 but it didn't let me choose that in the dropdown

**Comment by Totallynotmwa** [comment]
> it didn't let me choose that in the dropdown - @coffandro 

Idk how to do it on windows 11

**Comment by JayXTQ** [comment]
Windows 11 and 10 are very similar, I'm sure it's the same method.

**Comment by coffandro** [comment]
Looking through the web a lot of people are having this issue and the menu to do this has been removed in 11

**Comment by coffandro** [comment]
Because of this we can conclude that this is a win 11 issue rather than atlas

**Comment by Totallynotmwa** [comment]
> Because of this we can conclude that this is a win 11 issue rather than atlas- @coffandro 

Yep

**Comment by coffandro** [comment]
![image](https://github.com/Atlas-OS/Atlas/assets/75324436/74b7c24b-0ed6-4c01-ac95-62732c3a7222)
Nope, still hapening for some reason also, it dosen't register when trying to connect my RP2040-Zero to Thonny and when i press reboot the pc stays on that screen indefinetly?

**Comment by jacksyrm** [comment]
I'm closing this as we are releasing v0.3.1; re-open this if the issue continues on latest. Very unique and can't reproduce.

**Comment by coffandro** [comment]
Turns out to be a common issue with asus laptops https://www.reddit.com/r/ASUSROG/comments/11cx0i8/wifi_stops_working_after_turning_on_from_sleeprog/

**Comment by Louis047** [comment]
@coffandro i can confirm that, its has power-saving feature which causes this issue so i set my laptop to not sleep for my convenience.(My Laptop: ASUS TUF-A15 2021)

**Comment by coffandro** [comment]
Ah okay yeah that makes sense, I just use hibernate most of the time anyway and don't really need sleep tbh, so that should be fine

**Comment by Mudassir327** [comment]
I also faced the same issue on my ASUS laptop where the WiFi option kept disappearing.
Tried all the usual fixes (driver reinstall, updates, etc.) but nothing worked.

Finally, I opened the laptop, removed the WiFi card, cleaned its contacts gently with an eraser, and re-seated it back.
It’s been working perfectly without any issues for over a year now.

So, if drivers/updates don’t help, try checking the WiFi card connection itself.
